### PR TITLE
Base Quarkus Services

### PR DESCRIPTION
**Pull request by brunobat**
Please verify the names for group, artefacts and packages.

### CODE REVIEW

File: code/pom.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.azure.examples</groupId>
```
Comment (jeanbisutti) [inline code/pom.xml:7]: @SandraAhlgrimm Does it seem good for you?

File: code/quarkus-telemetry/quarkus-telemetry-superhero/src/test/java/com/azure/examples/quarkus/superhero/message/SuperHeroCreatorTest.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+import io.quarkus.test.junit.QuarkusTest;
+import org.junit.jupiter.api.Test;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+
+@QuarkusTest
+class SuperHeroCreatorTest {
```
Comment (jeanbisutti) [inline code/quarkus-telemetry/quarkus-telemetry-superhero/src/test/java/com/azure/examples/quarkus/superhero/message/SuperHeroCreatorTest.java:11]: We could perhaps remove the test folder to simplify the repo content.

File: code/quarkus-telemetry/quarkus-telemetry-legume/src/test/java/com/azure/examples/quarkus/LegumeResourceTest.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+import static com.github.tomakehurst.wiremock.client.WireMock.matching;
+import static com.github.tomakehurst.wiremock.client.WireMock.post;
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+
+@QuarkusTest
+public class LegumeResourceTest {
```
Comment (jeanbisutti) [inline code/quarkus-telemetry/quarkus-telemetry-legume/src/test/java/com/azure/examples/quarkus/LegumeResourceTest.java:21]: We could perhaps remove the test folder to simplify the repo content.

**Review by jeanbisutti — APPROVED** [review]

File: code/quarkus-telemetry/quarkus-telemetry-superhero/src/test/java/com/azure/examples/quarkus/superhero/message/SuperHeroCreatorTest.java (hunk 2 above)
Comment (brunobat) [reply code/quarkus-telemetry/quarkus-telemetry-superhero/src/test/java/com/azure/examples/quarkus/superhero/message/SuperHeroCreatorTest.java:11]: I'd prefer to keep the test, to make sure the code works... We should even add a native image test.

File: code/quarkus-telemetry/quarkus-telemetry-legume/src/test/java/com/azure/examples/quarkus/LegumeResourceTest.java (hunk 3 above)
Comment (brunobat) [reply code/quarkus-telemetry/quarkus-telemetry-legume/src/test/java/com/azure/examples/quarkus/LegumeResourceTest.java:21]: same as before.

**Pull request merged**